package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -537,254 +792,296 @@ def _update_water_selection(self):
-                                        (a.resname, a.resid, a.name),
-                                        dist, angle))
-
-            if water_bridges_acceptors:
-                self.logger_debug("Selection 2 Donors <-> Selection 2 Acceptors")
-                ns_acceptors = AtomNeighborSearch(water_bridges_acceptors)
-                for i, donor_h_set in self._s2_donors_h.items():
-                    d = self._s2_donors[i]
-                    for h in donor_h_set:
-                        res = ns_acceptors.search(h, self.distance)
-                        for a in res:
-                            donor_atom = h if self.distance_type != 'heavy' else d
-                            dist = distances.calc_bonds(donor_atom.position,
-                                                        a.position)
-                            if dist <= self.distance:
-                                angle = distances.calc_angles(d.position, h.position,
-                                                             a.position)
-                                angle = np.rad2deg(angle)
-                                if angle >= self.angle:
-                                    self.logger_debug(
-                                        "WB-A: {0!s} <-> S2-D: {1!s} {2:f} A, {3:f} DEG"\
-                                        .format(a.index, h.index, dist, angle))
-                                    s2_frame_results_dict[(a.resname, a.resid)].append(
-                                        (h.index, a.index,
-                                        (h.resname, h.resid, h.name),
-                                        (a.resname, a.resid, a.name),
-                                        dist, angle))
-
-            # Generate the water network
-            water_network = {}
-            for key in s2_frame_results_dict:
-                s1_frame_results = set(s1_frame_results_dict[key])
-                s2_frame_results = set(s2_frame_results_dict[key])
-                if len(s1_frame_results.union(s2_frame_results)) > 1:
-                    # Thus if selection 1 and selection 2 are the same and both
-                    # only form a single hydrogen bond with a water, this entry
-                    # won't be included.
-                    water_network[key] = [s1_frame_results,
-                    s2_frame_results.difference(s1_frame_results)]
-            # Generate frame_results
-            frame_results = []
-            for s1_frame_results, s2_frame_results in water_network.values():
-                frame_results.extend(list(s1_frame_results))
-                frame_results.extend(list(s2_frame_results))
-
-            self._timeseries.append(frame_results)
-            self._water_network.append(water_network)
-
-
-        logger.info("WBridge analysis: complete; timeseries  %s.timeseries",
-                    self.__class__.__name__)
     @staticmethod
     def _reformat_hb(hb, atomformat="{0[0]!s}{0[1]!s}:{0[2]!s}"):
```

Comment:
Does this work with the new output format?

Reply:
@orbeckst Yes, I'm only including it here so that it is compatible with hydrogen bond analysis.